compiler/liftdestructors.nim

```diff
@@ -600,7 +600,7 @@ proc createTypeBoundOps*(c: PContext; orig: PType; info: TLineInfo) =
   ## In the semantic pass this is called in strategic places
   ## to ensure we lift assignment, destructors and moves properly.
   ## The later 'injectdestructors' pass depends on it.
-  if orig == nil or {tfCheckedForDestructor, tfHasMeta} * orig.flags != {}: return
+  if orig == nil or {tfCheckedForDestructor, tfHasMeta} * orig.skipTypes({tyAlias}).flags != {}: return
```

Comment:
This one is extremely weird but helps fixing failing test. By the looks of it, i have digest collision in sighashes in code below. orig type is `distinct float` and canon type I get is `alias proxy`, very weird. Skipping tyAlias makes sure tyProxy doesn't get into sighashes table below.

This happens only in megatest and doesn't happen if you run tests individually.

orig type is `type USD = distinct float` from tests\concepts\tconcepts_issues.nim
canon type is `type VectorSpace[K] = concept x, y` from tests\concepts\tconcepts.nim

Reply:
Yeah megatest triggers quite some concept-related problems. Acceptable for now.